pkgs/development/libraries/smokegen/default.nix

```diff
@@ -0,0 +1,21 @@
+{ pkgs, lib, ... }:
```

Comment:
there's some issues in these 2 packages, i've fixed them in https://github.com/NixOS/nixpkgs/pull/205201